### PR TITLE
Replace got with powercord/http and eliminate random-useragent dependency

### PR DESCRIPTION
**Pull request by colin273**
node_modules has gone on a diet:

- I've replaced `@iamtraction/google-translate` with my own Powercord-ified [fork](https://github.com/FifiTheBulldog/google-translate-api-powercord), which uses `powercord/http` instead of `got` to make requests to Google Translate's API.
  - node_modules is just a shadow of its former self without `got` and all of its dependencies.
  - I removed token generation from the translation process. Apparently this step was broken all along, but the error was always caught silently, which made debugging way harder than it should have been. In any case, Google Translate doesn't seem to require the token anymore, and since generating a token requires a separate request to translate.google.com, eliminating this step means eliminating a useless request for every translation.
- `random-useragent` has been removed entirely, because the Google Translate module hasn't used the generated user agent since it replaced `translatte` last April.

Without the extra web request or generating a random user agent, translations seem to be running a lot faster in my (limited) testing. And there's only one tiny module in node_modules.

As an extra thing, I've changed the extension of style.scss to .css since it isn't using any SCSS-specific features, so it doesn't invoke the SCSS compiler unnecessarily on load.

### CODE REVIEW

File: TranslationHandler/translateAction.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-        let targetLanguage = userLang;
-        if (
-            Translator.cache[message.channel_id] &&
-            Translator.cache[message.channel_id][message.id] &&
-            Translator.cache[message.channel_id][message.id].currentLanguage != "original"
-        ) {
-            targetLanguage = "original";
-        }
-
-        Translator.translateMessage(
```
Comment (cyyynthia) [inline TranslationHandler/translateAction.js]: why did this get moved out of the if condition?

**Review by cyyynthia — CHANGES_REQUESTED** [review]
It appears your repository has a mismatched history which causes the PR to be marked as conflicting (probably due to the squash-and-merge done). They'll have to be solved for this PR to be merged.

To avoid these in the future I'd recommend in general making patches in a separate branch, which is created based on the upstream's latest commits.

Regardless, thanks a lot for the PR!

File: TranslationHandler/translateAction.js (hunk 1 above)
Comment (colin273) [reply TranslationHandler/translateAction.js]: I'm not sure, but unless I'm reading things wrong, I think that was that way before I started making changes.

Edit: indeed it was. https://github.com/cyyynthia/message-translate/blob/3a2839fdafc0569e983954b8a3d428ccbd202247/TranslationHandler/translateAction.js

**Comment by colin273** [comment]
All right, thanks for the tip! I'll keep that in mind for next time.

I think I've ironed out the merge conflicts now?

**Review by cyyynthia — APPROVED** [review]

**Pull request merged**